renovate.json

```diff
@@ -1,3 +1,42 @@
 {
-  "extends": ["github>ministryofjustice/arn-renovate-config"]
-}
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:node"],
+  "prBodyTemplate": "{{{table}}}{{{notes}}}{{{warnings}}}{{{controls}}}",
+  "packageRules": [
+    {
+      "matchManagers": ["npm"],
+      "rangeStrategy": "bump"
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "all non major NPM dependencies",
+      "groupSlug": "all-npm-minor-patch",
+      "stabilityDays": 5
+    },
+    {
+      "matchDepTypes": ["engines"],
+      "enabled": false
+    },
```

Comment:
This seems covered already in the base node config from hmpps: https://github.com/ministryofjustice/hmpps-renovate-config/blob/main/node.json#L6-L9